Sofa/framework/Core/src/sofa/core/topology/TopologySubsetData.inl

```diff
@@ -166,6 +166,9 @@ void TopologySubsetData <TopologyElementType, VecT>::remove(const sofa::type::ve
     // check for each element index to remove if it concern this subsetData
     for (Index elemId : index)
     {
+        if (data.size() == 0)
+            return;
```

Comment:
This check  should be move above the for-loop IMO
data will not change in the loop (and will return at the first iteration anyway if it is empty)

Reply:
in fact data is resized line:
https://github.com/sofa-framework/sofa/blob/83c4dbdfa46723270392d2e0818feb9496d98b24/Sofa/framework/Core/src/sofa/core/topology/TopologySubsetData.inl#L188-L189
this is why I have added this check.

Reply:
ah i did not see data was modified at the end of the loop 😅